includes/namespace.php

```diff
@@ -277,6 +279,83 @@ function deactivate( $network_wide ) {
 register_activation_hook( WEBSTORIES_PLUGIN_FILE, __NAMESPACE__ . '\activate' );
 register_deactivation_hook( WEBSTORIES_PLUGIN_FILE, __NAMESPACE__ . '\deactivate' );
 
+
+/**
+ * Append result of internal request to REST API for purpose of preloading data to be attached to a page.
+ * Expected to be called in the context of `array_reduce`.
+ *
+ * Like rest_preload_api_request() in core, but embeds links.
+ *
+ * @link  https://core.trac.wordpress.org/ticket/51636
```

Comment:
I think we should create a new core ticket for supporting embeds in this function

Reply:
I will create a new core ticket for this one.